fontforgeexe/wordlistparser.c

```diff
@@ -284,15 +284,21 @@ u_WordlistEscapedInputStringToRealString_readGlyphName(
 	    {
 		unichar_t* endptr = 0;
 		long unicodepoint = u_strtoul( glyphname+3, &endptr, 16 );
+                SplineChar* tmp = 0;
```

Comment:
Mmh... it will only matter on arch where NULL is not 0x0 and these [are rather old](http://stackoverflow.com/a/2597232). If it matters we should audit the code and try to remove all spots where that is (surely it's not only present there?).